### DISCUSSION TITLE
求助

### DISCUSSION THREAD

**Discussion by 985037983** [tw93/Pake#509] [Q&A]
Info Target: x64
     Running candle for "main.wxs"
     Running light to produce D:\a\Pake\Pake\node_modules\pake-cli\src-tauri\target\release\bundle\msi\94_1.0.0_x64_en-US.msi
       Error failed to bundle project: error running light.exe
file:///D:/a/Pake/Pake/node_modules/pake-cli/dist/cli.js:369
                reject(new Error(`${code}`));

**Comment by Tlntin** [comment]
日志不全，请提供全部日志。
